src/time/location.cr

```diff
@@ -299,7 +299,26 @@ class Time::Location
   # Files are cached based on the modification time, so subsequent request for
   # the same location name will most likely return the same instance of
   # `Location`, unless the time zone database has been updated in between.
+  #
+  # - `.load?` returns `nil` if the location is unavailable.
   def self.load(name : String) : Location
+    load?(name) { |source| raise InvalidLocationNameError.new(name, source) }
+  end
+
+  # :ditto:
+  #
+  # Returns `nil` if the location is unavailable.
+  # Raises `InvalidLocationNameError` if the name is invalid.
+  # Raises `InvalidTZDataError` if the loader encounters a format error in the
+  # time zone database.
+  #
+  # - `.load` raises if the location is unavailable.
```

Comment:
It's weird to get a bullet point about the behavior of another, alternative method, without any introduction to why we talk about it. For example "Consider X instead to raise when the location is unavailable" (same remark for `.load`).

Reply:
We use the same format in many other doc comments. I suppose we could introduce the list with "See also:"? But it feels odd to do this only a single instance. We should update the existing documentation as well, which makes it a different change.